pyproject.toml

```diff
@@ -114,6 +114,7 @@ ignore="H017"
 "network-api/networkapi/wagtailpages/templates/publication_child_menu.html" = "H006,H008,H013"
 "network-api/networkapi/wagtailpages/templates/sticky_dropdown.html" = "H006,H008,H013"
 "network-api/networkapi/wagtailpages/templates/wagtailpages/bannered_campaign_page.html" = "T003"
+"network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/about_block.html" = "H006"
```

Comment:
H006 error states "<img> must have weight and height defined", however, I am implementing the image in the same way that we are doing so on the [foundation site's homepage](https://github.com/MozillaFoundation/foundation.mozilla.org/blob/1af651384ac762253c9f8d12cc469acb10c7ab05/network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/quote.html#L13-L19), which is quite similar to this tickets design:

![Screenshot 2023-06-25 at 23-32-52 Mozilla Foundation](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/18314510/e801b9f4-5a3b-4a83-b877-e60af093f059)


Adding this error to the ignore list until we figure out a more uniform way to implement images!

Reply:
I've added a comment on the block's template.

Reply:
This also might be a good opportunity to ask @beccaklam, at the moment, the image is set to take up the width of the container and is **not** set to follow any rules regarding image height. 

This is the same as the quote section on the foundation homepage (screenshot above)

That means that if someone uploads a long image, the block will expand to fit it: 
![image](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/18314510/512cf883-a1d2-4c21-90ba-224f17ac324f)


Are there any dimensions that you would like this image to follow? 

Thanks!